### ISSUE TITLE
Feature request - implement ToString() overrides on complex types in ServerType

### ISSUE THREAD

**Issue by tonybaloney** [DimensionDataCBUSydney/DimensionData.ComputeClient#5]
The response of Get-CaasServers is not great to read because there are so many complex types in the field list.

We should implement `ToString` overrides to types like OperatingSystemType, ServerTypeNetworkInfo

```
operatingSystem : DD.CBU.Compute.Api.Contracts.Network20.OperatingSystemType
cpu             : DD.CBU.Compute.Api.Contracts.Network20.CpuType
memoryGb        : 4
disk            : {129135ed-96dd-48fe-90f1-60fa962c34ed}
Item            : DD.CBU.Compute.Api.Contracts.Network20.ServerTypeNetworkInfo
backup          : 
monitoring      : 
softwareLabel   : 
sourceImageId   : 0fcabe1a-df59-4dab-b552-050ba9dc675f
createTime      : 5/26/2016 2:07:34 AM
deployed        : True
started         : False
state           : NORMAL
progress        : 
vmwareTools     : DD.CBU.Compute.Api.Contracts.Network20.VmwareToolsType
virtualHardware : DD.CBU.Compute.Api.Contracts.Network20.VirtualHardwareType
id              : e7d756dd-084d-45ab-bd4d-241b85f156b4
datacenterId    : AU9
```

**Comment by tonybaloney** [comment]
Adam pointed me to a better way of doing this, there's an ps1format.xml convention document you can include as part of the module. Will research it.

**Comment by tonybaloney** [comment]
https://msdn.microsoft.com/en-us/library/gg574421(v=vs.85).aspx

**Comment by tonybaloney** [comment]
See #13

**Comment by tonybaloney** [comment]
closed in #13
